src/test/kotlin/org/objectionary/ddr/TestBase.kt

```diff
@@ -74,7 +84,29 @@ interface TestBase {
     /**
      * @return name of the test being executed
      */
-    fun getTestName() = Thread.currentThread().stackTrace[4].methodName
-        .substring(5)
-        .replace(' ', '_')
+    fun getTestName(): String? {
+        Thread.currentThread().stackTrace.forEach {
+            val methodName = it.methodName
+            if (methodName.substring(0, "test ".length) == "test ") {
+                return methodName.substring("test ".length).replace(' ', '_')
+            }
+        }
+        return null
```

Comment:
@c71n93 Maybe it would be better to throw an exception here? According to: https://www.yegor256.com/2014/05/13/why-null-is-bad.html